### PR TITLE
SOV-2955: update microservice for lend

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-2955

### CODE REVIEW

File: src/routes/lendingApy.route.ts [hunk 1]
```diff
@@ -33,3 +35,82 @@ router.get(
...
+  '/:lendingPool/daily',
+  checkBodyAndParams('lendingPool')
+    .exists()
+    .trim()
+    .isEthereumAddress()
+    .toLowerCase(),
+  setCacheMiddleware(30),
+  asyncMiddleware(async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      req.log.info(req, 'get daiy lending pool apy', [req.params.lendingPool])
```
Comment (soulBit) [inline src/routes/lendingApy.route.ts:51]:
```suggestion
      req.log.info(req, 'get daily lending pool apy', [req.params.lendingPool])
```

File: src/migration/1660550354504-TvlGroupEnum.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@ export class TvlGroupEnum1660550354504 implements MigrationInterface {
...
 
   public async up (queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query('ALTER TYPE "public"."tvl_tvlgroup_enum" RENAME TO "tvl_tvlgroup_enum_old"')
-    await queryRunner.query('CREATE TYPE "public"."tvl_tvlgroup_enum" AS ENUM(\'tvlAmm\', \'tvlLending\', \'tvlProtocol\', \'tvlSubprotocols\', \'tvlStaking\', \'tvlZero\')')
-    await queryRunner.query('ALTER TABLE "tvl" ALTER COLUMN "tvlGroup" TYPE "public"."tvl_tvlgroup_enum" USING "tvlGroup"::"text"::"public"."tvl_tvlgroup_enum"')
-    await queryRunner.query('DROP TYPE "public"."tvl_tvlgroup_enum_old"')
+    // await queryRunner.query('ALTER TYPE "public"."tvl_tvlgroup_enum" RENAME TO "tvl_tvlgroup_enum_old"')
+    // await queryRunner.query('CREATE TYPE "public"."tvl_tvlgroup_enum" AS ENUM(\'tvlAmm\', \'tvlLending\', \'tvlProtocol\', \'tvlSubprotocols\', \'tvlStaking\', \'tvlZero\')')
+    // await queryRunner.query('ALTER TABLE "tvl" ALTER COLUMN "tvlGroup" TYPE "public"."tvl_tvlgroup_enum" USING "tvlGroup"::"text"::"public"."tvl_tvlgroup_enum"')
+    // await queryRunner.query('DROP TYPE "public"."tvl_tvlgroup_enum_old"')
```
Comment (soulBit) [inline src/migration/1660550354504-TvlGroupEnum.ts:10]: We can remove this code if it's not going to be used in future, unless relevant for future reference (in which case you can close this comment)

**Review by soulBit — CHANGES_REQUESTED** [review]
Looks good, just two small comments to check

File: src/migration/1660550354504-TvlGroupEnum.ts (hunk 2 above)
Comment (creed-victor) [reply src/migration/1660550354504-TvlGroupEnum.ts]: Yep, should be uncommented

**Review by soulBit — APPROVED** [review]

**Pull request merged**